cypress/e2e/patient_spec/encouter-notes.cy.ts

```diff
@@ -0,0 +1,78 @@
+import { FacilityCreation } from "@/pageObject/facility/FacilityCreation";
+import { PatientEncounter } from "@/pageObject/Patients/PatientEncounter";
+
+const facilityCreation = new FacilityCreation();
+const patientEncounter = new PatientEncounter();
+
+describe("Encounter Notes - Threads, Messages, and Notes", () => {
+  beforeEach(() => {
+    cy.loginByApi("devdoctor"); // logging in as doctor
+    cy.visit("/");
+    facilityCreation.selectFacility("GHC Trikaripur");
```

Comment:
```suggestion
    facilityCreation.selectFacility("GHC Payyanur");
```

use the facility GHC Payyanur